net/munge/Portfile

```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
-# $Id$
```

Comment:
Yup, I'm already planning on doing that.  I wasn't sure if it was OK to squash into an already-submitted commit.